src/pages/MoviesView/Movies.jsx

```diff
@@ -18,20 +18,41 @@ const MoviesView = () => {
   const movies = useSelector(selectMovies);
   const isLoading = useSelector(selectLoading);
   const totalPages = useSelector(selectTotalPages);
-  const { page = 1, filter } = queryString.parse(location.search);
+  const {
+    page = 1,
+    filter,
+    year,
+    vote_average,
+    sortBy,
+  } = queryString.parse(location.search);
+  const { genre = [] } = queryString.parse(location.search);
```

Comment:
why you use the same line twice?
`= queryString.parse(location.search);`
`const { genre = [] } = queryString.parse(location.search);`